### PR TITLE
quickfix for nlb

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*
bug fix alb and nginx

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
       helm:
         releaseName: ingress-nginx-dev
         valueFiles:
-          - $values/packages/ingress-nginx/dev/values.yaml
+          - $values/packages/ingress-nginx/dev/values-dev.yaml
```
Comment (elamaran11) [inline platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml]: Where is this `values-dev.yaml` file?

File: platform/infra/terraform/deploy-apps/ingress-nginx-prod.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ spec:
       helm:
         releaseName: ingress-nginx-prod
         valueFiles:
-          - $values/packages/ingress-nginx/dev/values.yaml
+          - $values/packages/ingress-nginx/dev/values-prod.yaml
```
Comment (elamaran11) [inline platform/infra/terraform/deploy-apps/ingress-nginx-prod.yaml]: Where is this `values-prod.yaml` file?

File: platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml (hunk 1 above)
Comment (shapirov103) [reply platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml]: +1 to this comment. 

You may need to consider passing the parameter to override the name using helm parameter:

```
source:
  helm:
    parameters:
        - name: "contoller.service...."
          value: LoadBalancer
```

Comment (hmuthusamy) [reply platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml]: @shapirov103 @elamaran11  Changes have been done and fixed with new commit. Please merge as this is a breaking change

File: platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml [hunk 3]
```diff
@@ -11,11 +11,23 @@ spec:
       targetRevision: 4.7.0
       helm:
         releaseName: ingress-nginx-dev
-        valueFiles:
-          - $values/packages/ingress-nginx/dev/values.yaml
-    - repoURL: https://github.com/elamaran11/cnoe-appmod-implementation
-      targetRevision: feature/upgradeToPathBasedRouting
-      ref: values
+        parameters:
```
Comment (shapirov103) [inline platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml:14]: Can we keep values file and just override one parameter for the lb name?

Comment (shapirov103) [reply platform/infra/terraform/deploy-apps/ingress-nginx-dev.yaml:14]: deferring, need a clear approach that allows both values file and parameter override. 
Based on [this](https://argo-cd.readthedocs.io/en/latest/user-guide/helm/#helm-value-precedence)  I expected it work as designed, but could be a number of things, including defects, that prevent this behavior.

**Pull request merged**